### ISSUE TITLE
Markdown version of spec does not contain images

### ISSUE THREAD

**Issue by MartyIX** [Microsoft/TypeScript#5671]
http://typescriptlang.org links to https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md but this version does not contain any images. There are only slashes (`/`) instead.

An example:
```
A typical client would not need to add any additional typing but could just use a community-supplied typing to discover (through statement completion with documentation tips) and verify (through static checking) correct use of the library, as in the following screen shot.

/

Section [3.3](#3.3) provides additional information about object types.

## <a name="1.4"/>1.4 Structural Subtyping
```

**Comment by DanielRosenwasser** [comment]
There are PDF and DOCX versions which should have the images: https://github.com/Microsoft/TypeScript/tree/master/doc

I'm not sure if we'd be willing to take the time to fix up the markdown version for this, but perhaps if someone's interested in amending the script to do this sort of thing, that would be useful.

**Comment by MartyIX** [comment]
I think it is much faster to do this by hand. There are only four images that are missing and it's very easy to fix it.

**Comment by MartyIX** [comment]
I see. 

Well, I did an experiment https://github.com/MartyIX/TypeScriptDocExperiment/blob/master/index.js and it seems to work but it is very dummy.

**Comment by mhegazy** [comment]
here is the script that coverts the word document to md: https://github.com/Microsoft/TypeScript/blob/master/scripts/word2md.ts

and here is how it is called from jake: https://github.com/Microsoft/TypeScript/blob/master/Jakefile.js#L544

**Comment by DanielRosenwasser** [comment]
As a heads up, that script expects Word to be installed, and it either takes a while to process or I haven't been able to get it working.

**Comment by MartyIX** [comment]
I gave up after a few hours. I couldn't make it work on my computer. The program hangs but it does not crash on my computer. I waited a long time, I don't think it can take more than a few seconds when it works.

But I thought it was because I had Office 2013 that expired and I reverted back to Office 2007.

**Comment by DanielRosenwasser** [comment]
Yes, I've never been able to actually run it fully; I get the same problem. @ahejlsberg is there a secret to running this correctly?

**Comment by ahejlsberg** [comment]
Just tried it on my machine. It works, takes about 30 seconds to generate the spec.md file. You need to have Word installed. I'm using Word 2013 and I know it works with Word 2010 as well.

I know there are issues if you already have a Word process running when you start word2md, so make sure that isn't the case.

**Comment by mhegazy** [comment]
figured out the issue, the path to the spec file needs to be a full path. the jake command uses a full path. should be fixed in master now.

**Comment by DanielRosenwasser** [comment]
Technically a duplicate of #1566 and fixed by #6144.

**Comment by MartyIX** [comment]
:+1:
